io.greenfire.Greenery.json

```diff
@@ -0,0 +1,87 @@
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "greenery",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 io.greenfire.Greenery.desktop /app/share/applications/io.greenfire.Greenery.desktop",
+                "install -Dm644 io.greenfire.Greenery.appdata.xml /app/share/appdata/io.greenfire.Greenery.appdata.xml",
+                "install -Dm644 -t /app/share/icons/hicolor/scalable/apps io.greenfire.Greenery.svg",
+                "install -Dm644 io.greenfire.Greenery-128x128.png /app/share/icons/hicolor/128x128/apps/io.greenfire.Greenery.png",
+                "install -Dm644 io.greenfire.Greenery-64x64.png /app/share/icons/hicolor/64x64/apps/io.greenfire.Greenery.png",
+                "install -D apply_extra /app/bin/apply_extra",
+                "install -D greenery /app/bin/greenery"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "unzip greenery.zip > /dev/null",
+                        "mv Greenery-linux-x64/* .",
+                        "rm -r Greenery-linux-x64 *.zip"
+                    ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "greenery",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
+                        "exec zypak-wrapper /app/extra/greenery \"$@\""
+                    ]
+                },
+                {
+                    "type": "extra-data",
+                    "url": "https://github.com/GreenfireInc/Releases.Greenery/releases/download/v0.9.7/Greenery-linux-x64-0.9.7.zip",
+                    "sha256": "b230efd545589effebdade8f20162801b71733f2f267c1f9a2ad7e285bc042fd",
+                    "size": 221597803,
+                    "filename": "greenery.zip",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "https://github.com/GreenfireInc/Releases.Greenery/releases",
+                        "version-pattern": "https://github.com/GreenfireInc/Releases.Greenery/releases/hashes-Greenery.([\\d\\.-]+).txt",
+                        "url-template": "https://github.com/GreenfireInc/Releases.Greenery/releases/Greenery-linux-x64.$version.zip"
+                    }
+                },
+                {
+                    "type": "file",
+                    "path": "io.greenfire.Greenery.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "io.greenfire.Greenery-128x128.png"
+                },
+                {
+                    "type": "file",
+                    "path": "io.greenfire.Greenery-64x64.png"
+                },
```

Comment:
```suggestion
```

that was implied.

Reply:
Yes I see this issue! I have removed the references to the png images in the area highlighted and also in the build-commands.  Pushing up update.